templates/foreman_memcache.yaml.erb

```diff
@@ -0,0 +1,11 @@
+# Foreman Memcache plugin configuration file
+
+:memcache:
+  :hosts:
+<% @hosts.each do |host| -%>
+    - <%= host %>
+<% end -%>
+  :options:
+    :namespace: <%= scope.lookupvar("::foreman::plugin::memcache::namespace") %>
```

Comment:
`<%= @namespace %>` because the namespace variable is in the same scope as the `template()` call in memcache.pp.  Same goes for expires_in and compress.